frameworks/kafka/universe/package.json

```diff
@@ -7,21 +7,22 @@
   "downgradesTo": [
     "{{downgrades-to}}"
   ],
-  "name": "kafka",
+  "name": "portworx-kafka",
   "version": "{{package-version}}",
-  "maintainer": "support@mesosphere.io",
+  "maintainer": "support@portworx.com",
   "description": "Apache Kafka is used for building real-time data pipelines and streaming apps. It is horizontally scalable, fault-tolerant, wicked fast, and runs in production in thousands of companies.",
   "selected": true,
```

Comment:
selected should be false
Check all the fields from current portworx-kafka in dcos-commons and make sure the values match.

Reply:
Changes added.